src/docs/sphinx/dev_guide/gpu_porting.rst

```diff
@@ -161,6 +160,199 @@ GPU device, or on both a GPU device and a CPU host. For example::
   When Axom is built without RAJA, ``axom::for_all`` becomes a ``for``-loop on
   host (CPU).
 
+%%%%%%%%%%%%%%%%
+Portability
+%%%%%%%%%%%%%%%%
```

Comment:
I really appreciate this whole section, Brad.

Reply:
Thanks. I wanted to note these nvcc pitfalls since I did not know about them before I was hit by them.